+euphonic/install_python_modules.m

```diff
@@ -0,0 +1,21 @@
+function install_python_modules()
+    try
+        req_mods = required_modules;
+        verify_python_modules(req_mods{:});
+    catch ME
+        % Checks we have the right Python version (>=3.6)
+        if ~strcmp(ME.identifier, 'euphonic_horace_interface:verify_python_modules:pythonVersion')
+            rethrow(ME);
+        end
+    end
+    pipe = py.subprocess.PIPE;
+    kwargs = pyargs('stdout', pipe, 'stderr', pipe);
+    out = py.subprocess.run({py.sys.executable '-m' 'pip' 'install' 'euphonic' 'horace-euphonic-interface'}, kwargs);
```

Comment:
This will just install the latest euphonic, we should probably install the recommended version from `required_modules`